package.json

```diff
@@ -102,7 +104,10 @@
     },
     "dependencies": {
         "@faker-js/faker": "^9.8.0",
-        "@tool-belt/type-predicates": "^1.4.0"
+        "@jest/globals": "30.0.0-beta.3",
```

Comment:
ok, self revierw this PR before I continue - clean it completely from everything out of scope.